### PR TITLE
Zth setup

### PR DESCRIPTION
**Pull request by zachholcomb**
Just a couple formatting things
Only big change was to change game_teams to game_team per what we talked about in slack
Also fixed the game class to change the incoming information into integers

### CODE REVIEW

File: lib/game.rb [hunk 1]
```diff
@@ -10,14 +10,14 @@ class Game
               :venue
 
   def initialize(info_params)
-    @game_id = info_params[:game_id]
-    @season = info_params[:season]
+    @game_id = info_params[:game_id].to_i
```
Comment (zachholcomb) [inline lib/game.rb:13]: Forgot these in my first draft of this class, we'll need these later

File: test/game_team_test.rb [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+      shots: "8",
+      tackles: "44",
+      pim: "8",
+      power_play_opps: "3",
+      power_play_goals: "0",
+      face_off_win_pct: "44.8",
+      giveaways: "17",
+      takeaways: "7"
+    }
+    @game_team3 = GameTeam.new(@game)
```
Comment (zachholcomb) [inline test/game_team_test.rb:24]: Also added game_team3 to setup, super minor though, it could be either way

File: test/game_test.rb [hunk 3]
```diff
@@ -4,14 +4,14 @@
...
-            season: 20122013,
+            game_id: '2012030221',
+            season: '20122013',
             type: 'Postseason',
             date_time: '5/16/13',
-            away_team_id: 3,
-            home_team_id: 6,
-            away_goals: 2,
-            home_goals: 3,
+            away_team_id: '3',
```
Comment (zachholcomb) [inline test/game_test.rb:11]: made these all strings to test that they were being converted to integers where I needed them to

File: lib/game_team.rb [hunk 4]
```diff
@@ -1,4 +1,4 @@
-class GameTeams
+class GameTeam
```
Comment (elguapogordo) [inline lib/game_team.rb:1]: much gooder!

**Pull request merged**